src/inventory_ui.cpp

```diff
@@ -3525,7 +3531,9 @@ void inventory_multiselector::on_input( const inventory_input &input )
 
     if( input.entry != nullptr ) { // Single Item from mouse
         highlight( input.entry->any_item() );
-        toggle_entries( count );
+        if( input.action == "SELECT" ) {
+            toggle_entries( count );
+        }
     } else if( input.action == "TOGGLE_NON_FAVORITE" ) {
```

Comment:
```suggestion
    }
    if( input.action == "TOGGLE_NON_FAVORITE" ) {
```
This should make clicking on chevron (from #64750) work for multiselectors too